### ISSUE TITLE
Value `any` in request body

### ISSUE THREAD

**Issue by TBurda** [PaloAltoNetworks/docusaurus-openapi-docs#492]
## Describe the bug

There are no properties in the request body. Instead, the value `any` is displayed. Problem has occurred since version v1.6.1.

![Snímek obrazovky 2023-03-13 v 12 47 21](https://user-images.githubusercontent.com/21170562/224693488-b5992ceb-d469-481e-9ee1-ee6b6c215642.png)

```yaml
x-generator: NSwag v13.18.2.0 (NJsonSchema v10.8.0.0 (Newtonsoft.Json v13.0.0.0))
openapi: 3.0.0
info:
  title: Logeto public API
  version: 2.0.0
servers:
  - url: https://[AccountName].logeto.com
paths:
  /api/v2/Activities:
    post:
      tags:
        - Activities
      summary: Save new activity
      operationId: Activities_Post
      requestBody:
        x-name: request
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/ActivityRequest'
        required: true
        x-position: 1
      responses:
        '201':
          description: Successfully Inserted
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/SystemFieldsPostResponse'
        '400':
          description: Error
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/ErrorResponse'
components:
  schemas:
    ActivityType:
      type: string
      description: ''
      x-enumNames:
        - Work
        - Absence
        - Break
      enum:
        - Work
        - Absence
        - Break
    ActivityTypeTimeEntry:
      type: string
      description: ''
      x-enumNames:
        - Optional
        - TotalTime
        - FromTo
      enum:
        - Optional
        - TotalTime
        - FromTo
    SystemFieldsResponseBase:
      type: object
      x-abstract: true
      additionalProperties: false
    ErrorResponse:
      type: object
      additionalProperties: false
      required:
        - Error
      properties:
        Error:
          $ref: '#/components/schemas/ErrorBase'
    ErrorBase:
      type: object
      additionalProperties: false
      properties:
        Code:
          type: string
          nullable: true
        Message:
          type: string
          nullable: true
    SystemFieldsPostResponse:
      allOf:
        - $ref: '#/components/schemas/SystemFieldsResponseBase'
        - type: object
          additionalProperties: false
          properties:
            Guid:
              type: string
              format: guid
              nullable: true
    ActivityRequest:
      allOf:
        - $ref: '#/components/schemas/SystemFieldsPostRequest'
        - type: object
          additionalProperties: false
          required:
            - Name
          properties:
            Name:
              type: string
              maxLength: 100
              minLength: 1
            Code:
              type: string
              maxLength: 12
              nullable: true
            Type:
              default: Work
              nullable: true
              oneOf:
                - $ref: '#/components/schemas/ActivityType'
            TimeEntry:
              default: Optional
              nullable: true
              oneOf:
                - $ref: '#/components/schemas/ActivityTypeTimeEntry'
            Inactive:
              type: boolean
              nullable: true
            Movement:
              type: boolean
              nullable: true
            Icon:
              type: string
              nullable: true
    SystemFieldsPostRequest:
      type: object
      additionalProperties: false
  securitySchemes:
    AccessKey:
      type: apiKey
      name: AccessKey
      in: header
```

Swagger editor will display request body correctly.

![Snímek obrazovky 2023-03-13 v 12 48 08](https://user-images.githubusercontent.com/21170562/224693630-4dc66711-f6e2-4921-a343-259fda799864.png)

## Your Environment

- Version used: v2.0.0-beta.1

**Comment by sserrata** [comment]
Hi @TBurda, this is definitely a regression bug introdudced in #465. Can you have a look below to confirm this is what you would expect? 

<img width="962" alt="Screenshot 2023-03-13 at 10 17 36 AM" src="https://user-images.githubusercontent.com/9343811/224728436-54888ff5-0873-4e56-b93d-aba5c76c7dc0.png">

**Comment by sserrata** [comment]
Fix now available to test in canary `0.0.0-596`

**Comment by sserrata** [comment]
Fix will be in next release, thanks!
